eq-author/src/App/page/Design/answers/MultipleChoiceAnswer/index.js

```diff
@@ -137,7 +137,7 @@ export class UnwrappedMultipleChoiceAnswer extends Component {
         type={answer.type}
       >
         <AnswerWrapper>
-          {answer.type === CHECKBOX && (
+          {answer.type === CHECKBOX && answer.label === "" && (
             <AnswerHelper>Select all that apply</AnswerHelper>
```

Comment:
I think the card suggest completely removing 'select all that apply' on design page.

Reply:
Yes, checked with Sarah, have removed the AnswerHelper block.